src/Cli/dotnet/OptionForwardingExtensions.cs

```diff
@@ -44,6 +44,12 @@ public static Option<T> AllowSingleArgPerToken<T>(this Option<T> option)
             return option;
         }
 
+        public static Option<T> MakeHidden<T>(this Option<T> option)
```

Comment:
Change looks good, wondering if we should call this Hide instead

Reply:
Sure, just renamed.